test_output/test_assembly.wdl

```diff
@@ -1,46 +1,55 @@
+version 1.0
 import "jgi_assembly.wdl" as jgi
 
 workflow test_assembly {
+  input{
   String bbtools_container="microbiomedata/bbtools:38.94"
   String spades_container="microbiomedata/spades:3.15.0"
   String validate_container="mbabinski17/comparejson:0.1"
   String rename_contig_prefix="scaffold"
-  Float  uniquekmer=1000
-  String? memory="60G"
-  String? threads="8"
-  String? outdir="/vol_b/nmdc_workflows/test_nmdc/metaAssembly/test_output"
-  String  url="https://portal.nersc.gov/cfs/m3408/test_data/Ecoli_10x-int.fastq.gz"
-  String  ref_json="https://raw.githubusercontent.com/microbiomedata/metaAssembly/master/test_output/small_test_stats.json"
-
+  # Float  uniquekmer=1000
+  String proj
+  String memory="60G"
+  String threads="8"
+  # String outdir="/vol_b/nmdc_workflows/test_nmdc/metaAssembly/test_output"
+  String url="https://portal.nersc.gov/cfs/m3408/test_data/Ecoli_10x-int.fastq.gz"
+  String ref_json="https://raw.githubusercontent.com/microbiomedata/metaAssembly/master/test_output/small_test_stats.json"
+  }
   call prepare {
-    input: container=bbtools_container,
-	   url=url,
-           ref_json=ref_json
+    input: 
+    container=bbtools_container,
+	  url=url,
+    ref_json=ref_json
   }
   call jgi.jgi_metaASM as asm {
-    input: input_file=[prepare.fastq],
-           bbtools_container=bbtools_container,
-	   spades_container=spades_container,
-           memory=memory,
-           threads=threads,
-	   uniquekmer=uniquekmer,
-	   rename_contig_prefix=rename_contig_prefix,
-	   outdir=outdir
+    input: 
+    input_file=prepare.fastq,
+    proj = proj,
+    bbtools_container=bbtools_container,
+	  spades_container=spades_container,
+    memory=memory,
+    threads=threads,
+	  # uniquekmer=uniquekmer,
+	  rename_contig_prefix=rename_contig_prefix#,
+    # outdir=outdir
   }
   call validate {
-    input: container=validate_container,
-           refjson=prepare.refjson,
-           user_json=asm.final_asmstat
+    input: 
+    container=validate_container,
+    refjson=prepare.refjson,
+    user_json=asm.asmstats
   }
 }
 task prepare {
+  input{
    String container
    String ref_json
    String url
-   command{
-       wget -O "input.fastq.gz" ${url}
-       wget -O "ref_json.json" ${ref_json}
    }
+   command <<<
```

Comment:
same here all tasks should have 'set -euo pipefail' if the command block contains more than on bash command.

Reply:
This command block still needs a pipefail.